webpack.config.js

```diff
@@ -31,7 +34,17 @@ const config = {
       {
         test: /\.ts$/,
         exclude: /node_modules/,
-        use: ['ts-loader'],
+        use: [
+          {
+            loader: 'ts-loader',
+            options: {
+              transpileOnly: true,
```

Comment:
It says [here](https://github.com/TypeStrong/ts-loader#faster-builds) that `transpileOnly` will disable type checking... is this true?

Maybe we should make use of https://github.com/TypeStrong/fork-ts-checker-webpack-plugin

Reply:
Yes, and current gulp setup already have separate tsc runner I don't think webpack need duplicated type checking.

Reply:
Yeah, but it doesn't seem like any type checking happens on, say, `vsce package`. I'm a bit skeptical about that, even though it would fail the tests, etc. @jpoon any thoughts?

Reply:
There are lot of debates of webpack need to take care of type checking, in my opinion 1. Gulp build command already have seperate typecheck 2. And it should catch build failure instead of webpack 3. Webpack does typecheck only will increase build time by doing redundant type check.

Reply:
We definitely want type-checking. Right now we do it in parallel on build: 

```
gulp.task('build', gulp.series('prettier', gulp.parallel('webpack', 'tslint'), 'commit-hash'));
```

It's not clear to me if we have any value-add for moving to doing this with webpack.

Reply:
Ah, you're right. If we're explicitly doing `tslint` alongside webpack, I think the proposed change makes sense.